ballerina/connection-pool.bal

```diff
@@ -20,17 +20,17 @@ configurable int maxOpenConnections = 15;
 configurable decimal maxConnectionLifeTime = 1800.0;
 configurable int minIdleConnections = 15;
 
-# Represents the properties which are used to configure DB connection pool.
+# Represents the properties which are used to configure a DB connection pool.
 # Default values of the fields can be set through the configuration API.
 #
-# + maxOpenConnections - The maximum number of open connections that the pool is allowed to have including
-#                        both idle and in-use connections. The default value is 15. This can be changed through the
-#                        configuration API with the `ballerina.sql.maxOpenConnections` key
+# + maxOpenConnections - The maximum number of open connections that the pool is allowed to have.
```

Comment:
```suggestion
# + maxOpenConnections - The maximum number of open connections that the pool is allowed to have
```
Please check the guidelines.if we need this fullstop or not.

Reply:
We need a fullstop as the description for the parameter is multiple sentences.

Reply:
Isn't this the only sentence we have in the description? `The maximum number of open connections that the pool is allowed to have.`